README.md

```diff
@@ -1,9 +1,54 @@
-# scrappy
+# Scrappy the Slack Bot
```

Comment:
```suggestion
# Scrappy
```
I'm leaning towards keeping it at Scrappy but idk tbh

Reply:
Yeah that works for me!